src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -255,41 +255,62 @@ function MoneyRequestView({
         [transactionAmount, isSettled, isCancelled, isPolicyExpenseChat, isEmptyMerchant, transactionDate, hasErrors, canUseViolations, hasViolations, translate, getViolationsForField],
     );
 
+    let errors = {};
+    if (transaction?.errors) {
```

Comment:
We don't need extra if statements here, we can just use spread operators to reduce the statements.
@kaushiktd

Reply:
If we remove extra if statement, it will fail when submitted PR every time caused by TypeScript Checks by git.

Reply:
you can try nil collestion operator by using `...transaction?.errors ?? {}` or something similar

@kaushiktd

Reply:
@abdulrahuman5196 Resolved.